web/source/kmwembedded.ts

```diff
@@ -9,9 +12,173 @@
 /*                                       */
 /*****************************************/
 
+namespace com.keyman.osk {
+    // Send the subkey array to iOS, with centre,top of base key position
```

Comment:
Does this section belong in web/source/osk/?

Reply:
It parallels kmwnative.ts, which is also in the base directory instead of web/source/osk.

Both files override, extend, or re-implement specific parts of _all_ of KMW to better suit the operating environment for the target.  There's more than just OSK extension at play.

Maybe parts like this section could be in separate files - so /osk/embedded.ts, linked by `<reference path>` from the current kmwembedded.ts, with a /osk/native.ts parallel for kmwnative.ts's matching section.  Either way, it's very useful to have a single file inclusion for linking specific 'native' vs 'embedded' code for each target.